Examples/UIComponentExample/Examples/Complex layout/Components/IntroductionCard.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  IntroductionCard.swift
+//  UIComponentExample
+//
+//  Created by y H on 2021/7/22.
+//  Copyright © 2021 Luke Zhao. All rights reserved.
+//
+
+import UIKit
+import UIComponent
+
+struct IntroductionCard: ComponentBuilder {
+  private let longText = """
+    Write UI in crazy speed, with great perf & no limitations.
+    SwiftUI still haven't satisfied my requirements. So I build this.
+    
+    This framework allows you to build UI using UIKit with syntax similar to SwiftUI. You can think about this as an improved UICollectionView.
+    
+    Great performance through global cell reuse.
+    Built in layouts including Stack, Flow, & Waterfall.
+    Declaritive API based on resultBuilder and modifier syntax.
+    Work seemless with existing UIKit views, viewControllers, and transitions.
+    dynamicMemberLookup support for all ViewComponents which can help you easily update your UIKit views.
+    Animator API to apply animations when cells are being moved, updated, inserted, or deleted.
+    Simple architecture for anyone to be able to understand.
+    Easy to create your own Components.
+    No state management or two-way binding.
+    """
+  
+  
+  let expand: Bool
+  let tapHandler: (Bool) -> Void
```

Comment:
can probably remove the `(Bool)` parameter.

Reply:
✅